src/SapNwRfc/Internal/OutputMapper.cs

```diff
@@ -120,10 +125,21 @@ public static TOutput Extract<TOutput>(RfcInterop interop, IntPtr dataHandle)
             // ReSharper disable once PossibleNullReferenceException
             PropertyInfo fieldValueProperty = extractMethod.ReturnType.GetProperty(nameof(Field<object>.Value));
 
+            Collection<Expression> arguments = new Collection<Expression>() { interop, dataHandle, name };
+
+            if (propertyInfo.PropertyType == typeof(byte[]))
```

Comment:
Avoid testing the same condition twice. I would integrate this into the `if` block at line 94. (which means the arguments collection need to be defined before line 74.